File: modelAPI/Baidu/ERNIE_Bot.py
from .BaiduAPI import BaiduAPI

class ERNIE_Bot(BaiduAPI):
    def __init__(self):
        super().__init__()
        self.url = "https://aip.baidubce.com/rpc/2.0/ai_custom/v1/wenxinworkshop/chat/completions?access_token=" + self.access_token
    

File: modelAPI/Baidu/ERNIE_Bot.go
package Baidu

const ErnieBoturl = "https://aip.baidubce.com/rpc/2.0/ai_custom/v1/wenxinworkshop/chat/completions?access_token="

type ErnieBot struct {
	Baidu
	url string `https://aip.baidubce.com/rpc/2.0/ai_custom/v1/wenxinworkshop/chat/completions?access_token=`
}

func NewErnieBot() *ErnieBot {
	baidu := NewBaidu()
	accessToken, _ := baidu.getAccessToken()
	baidu.url = ErnieBoturl + accessToken
	ernieBot := &ErnieBot{
		Baidu: *baidu,
	}
	return ernieBot
}
